components/imageFormField.js

```diff
@@ -0,0 +1,119 @@
+import React, { useState, useEffect } from 'react';
+import { View, Text, Alert, Image, Platform } from 'react-native';
+import { Button } from 'react-native-elements';
+import * as ImagePicker from 'expo-image-picker';
+import * as Permissions from 'expo-permissions';
+import { styles } from './imageFormField.styles';
+import Icon from 'react-native-vector-icons/SimpleLineIcons';
+
+
+export const ImageCaptureFormField = () => {
+  const [capturedImage, setCapturedImage] = useState(null);
+
+  const verifyPermissions = async () => {
+    const result = await Permissions.askAsync(Permissions.CAMERA_ROLL);
+    if (result.status != 'granted') {
+      Alert.alert('You need to grant access to camera',
+        [{text: 'Okay'}]
+      );
+      return false;
+    }
+    return true;
+  };
+
+  const captureImageHandler = async () => {
+    const hasPermission = await verifyPermissions();
+    if(!hasPermission) {
+      return;
+    }
+    const image = await ImagePicker.launchCameraAsync({
+      allowsEditing: true,
+      aspect: [16, 9],
```

Comment:
super detail (probably for later): we could extract this to a config variable or file and reuse it also for below (picklibraryimage).